src/App.asm

```diff
@@ -1,6 +1,69 @@
 .data
 
+str1: .asciiz "Enter total number of students:"
+str2: .asciiz "Enter student score:"
+str3: .asciiz "Highest score is "
+count: .word 0
+i: .word 0
+top_student: .word -1
+score: .word 0
 
 .globl main
 
-.text
\ No newline at end of file
+.text
+
+main:
+
+la $a0, str1
+li $v0, 4
+syscall
+
+li $v0,5
+syscall
+move $t0, $v0
+sw $t0,count
+
+li $t6,1
+lw $t1, i
+lw $t2, top_student
+lw $t3, score
+
+
+begine_loop:
+ blt $t1,$t0, body_loop
```

Comment:
variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, body_loop
...
```